.github/workflows/build-electron.yml

```diff
@@ -94,10 +94,6 @@ jobs:
         DEBUG: electron-builder
       run: npm run dist
 
-      # Also run all tests on the build dir code?
-    - name: Test flask app binaries
-      run: npm run test-flask-app
```

Comment:
Do we still want this deleted?

Reply:
Yeah these tests now run as part of `npm run test`.